platforms/nuget/build-jinja.py

```diff
@@ -1,32 +1,78 @@
-import os
+import os, argparse
```

Comment:
add small docstring explaining the purpose of this script.

Reply:
I have added the docstring in our build python file now.